src/server/mailgun.ts

```diff
@@ -29,6 +34,16 @@ export async function mailGunConfig(): Promise<[null | ReturnType<Mailgun['clien
     return [_mg, domain]
 }
 
+const getStudyAndOrg = async (studyId: string) => {
+    const study = await getStudyAction(studyId)
+    // TODO DISCUSS in general, do we want to throw? or return undefined if no study found inside getStudyAction?
+    if (!study) throw new Error('Study not found')
+    const org = await getOrgFromIdAction(study.orgId)
+    if (!org) throw new Error('Org not found')
+
+    return { study, org }
+}
```

Comment:
I'm conflicted about:
 * do we want to call actions from server-side code?  While technically we can, they do have a bit more overhead because of the wrappers
 * Calling two functions that each run db queries seems inefficient, instead we should make a single query (maybe in db/queries?) to fetch the study and organization information at once.

Reply:
👍 makes sense